src/components/Heading/Heading.tsx

```diff
@@ -17,6 +17,18 @@ export const VARIANTS = [
 ] as const;
 export type Variant = typeof VARIANTS[number];
 export type HeadingElement = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+// For now, "h1"-"h6" sizes point to the old type ramp, while
+// design token sizes point to the new type ramp.
+// These will be brought in sync with the next major release.
+const TOKEN_TO_SIZE = {
+  'headline-lg': 'h1',
+  'headline-md': 'h2',
+  'headline-sm': 'h3',
+  'title-md': 'h4',
+  'title-sm': 'h5',
+  'body-sm': 'h6',
+};
+export type HeadingSize = HeadingElement | keyof typeof TOKEN_TO_SIZE;
```

Comment:
Was this stuff in a different PR? This looks really familiar..

Reply:
haha yeah I _just_ made this change on main another day